spring-data-jdbc-plus-sql/src/main/java/com/navercorp/spring/data/jdbc/plus/sql/convert/AggregateResultJdbcConverter.java

```diff
@@ -543,24 +543,6 @@ private boolean isSimpleProperty(RelationalPersistentProperty property) {
 			&& !property.isEmbedded();
 	}
 
-	/**
-	 * The interface Map parameter value provider.
-	 *
-	 * @param <P> the type parameter
-	 * @deprecated no usage in spring-jdbc-plus, to be removed.
-	 */
-	@Deprecated(since = "3.5.0", forRemoval = true)
-	protected interface MapParameterValueProvider<P extends PersistentProperty<P>> {
-		/**
-		 * Gets parameter value.
-		 *
-		 * @param parameter the parameter
-		 * @return the parameter value
-		 */
-		@Nullable
-		Object getParameterValue(Parameter<?, P> parameter);
-	}
```

Comment:
I've removed this code as it was marked @Deprecated since version 3.5.0, and we have now moved past it.

I don't think this is needed anymore, but I'm not 100% sure. Can anyone confirm it's safe to delete?

Reply:
I think deleting this interface at 3.7.0 will reduce the confusion a little more

Reply:
@IAM20 

Thanks for the feedback! I've reverted the change.